### PR TITLE
Style and clean GWTP Sample Basic Spring

### PR DESCRIPTION
**Pull request by jasonlemay**

### CODE REVIEW

**Comment by Chris-V** [comment]
LGTM

File: gwtp-samples/gwtp-sample-basic-spring/src/main/java/com/gwtplatform/samples/basicspring/client/resources/AppResources.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.gwtplatform.samples.basicspring.client.resources;
```
Comment (christiangoudreau) [inline gwtp-samples/gwtp-sample-basic-spring/src/main/java/com/gwtplatform/samples/basicspring/client/resources/AppResources.java]: missing license header

File: gwtp-samples/gwtp-sample-basic-spring/src/main/java/com/gwtplatform/samples/basicspring/client/resources/ResourceLoader.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.gwtplatform.samples.basicspring.client.resources;
```
Comment (christiangoudreau) [inline gwtp-samples/gwtp-sample-basic-spring/src/main/java/com/gwtplatform/samples/basicspring/client/resources/ResourceLoader.java]: missing license header

**Comment by christiangoudreau** [comment]
LGTM

File: gwtp-samples/gwtp-sample-basic-spring/src/main/resources/com/gwtplatform/samples/basicspring/client/resources/css/normalize.gss [hunk 3]
```diff
@@ -0,0 +1,425 @@
+/*! normalize.css v3.0.1 | MIT License | git.io/normalize */
```
Comment (Chris-V) [inline gwtp-samples/gwtp-sample-basic-spring/src/main/resources/com/gwtplatform/samples/basicspring/client/resources/css/normalize.gss]: MIT license. Samples are under Apache 2. Do they play well together?

Comment (christiangoudreau) [inline gwtp-samples/gwtp-sample-basic-spring/src/main/resources/com/gwtplatform/samples/basicspring/client/resources/css/normalize.gss]: It's a file by file basis, so no worries there. And there's not much difference between both

**Pull request merged**